nuget/roundhouse/roundhouse.nuspec

```diff
@@ -12,6 +12,15 @@
     <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <tags>roundhouse db migration database migrator chucknorris</tags>
-	<iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
+    <dependencies />
+	  <iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
   </metadata>
+  <files>
+    <file src="$mergedExe$" target="bin\" />
+    <file src="$appConfigFile$" target="bin\" />

```

Comment:
Ouch, we need a config file now?

Reply:
Maybe not? Looks like there is mostly bollocks in it. It was in rh 0.8.8 (the latest published one). I can leave it out, if we don't want it.